packages/cli/src/services/provider-service.ts

```diff
@@ -16,6 +16,11 @@ export function assertNameIsCorrect(name: string): void {
     throw new Error(`Project name cannot contain spaces:
 
     Found: '${name}'`)
+  
+  if (name.includes('_'))
+    throw new Error(`Project name cannot contain underscore(_):
```

Comment:
I think we can omit the parentheses 😄 

```suggestion
    throw new Error(`Project name cannot contain underscore:
```